src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -2774,7 +2774,12 @@ namespace ZoneEquipmentManager {
 
 					// initialize sizing conditions if they have not been set (i.e., no corresponding load) to zone condition
 					if ( FinalZoneSizing( CtrlZoneNum ).ZoneTempAtCoolPeak == 0.0 ) {
-						FinalZoneSizing( CtrlZoneNum ).ZoneTempAtCoolPeak = ZoneSizing( DDNumF, CtrlZoneNum ).DesCoolSetPtSeq( TimeStepAtPeakF );
```

Comment:
Here's where zone Tstat temps are used, min for cooling, max for heating. @mbadams5 suggested to protect for empty arrays (#6006) and I see no reason not to. If the arrays are never empty its no harm.